README.md

```diff
@@ -1 +1,54 @@
-# Global-racing-event
\ No newline at end of file
+# Global-racing-event
+# Global-racing-event
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Super SportCar Race:
+
+> Web page to Showcase a Fictional Super SportCar Race as Microverse Capstone Project.
+
+![screenshot](/Assets/img/screenshot.png)
```

Comment:
- [ ] The image you choose is bigger than required, I kindly invite you to check that, to have an image of this size could be annoying to the person who checked your project. 🐸
- I add a recommendation (ok well, two recommendations).
![image](https://user-images.githubusercontent.com/20563252/124698902-6f20dd80-deaf-11eb-830e-06c639f0be50.png)
![image](https://user-images.githubusercontent.com/20563252/124698928-7811af00-deaf-11eb-8db7-2b1763560011.png)